includes/Modules/Search_Console.php

```diff
@@ -527,4 +530,31 @@ protected function setup_services( Google_Site_Kit_Client $client ) {
 	protected function setup_settings() {
 		return new Settings( $this->options );
 	}
+
+	/**
+	 * Sets up the module's assets to register.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return Asset[] List of Asset objects.
+	 */
+	protected function setup_assets() {
+		$base_url = $this->context->url( 'dist/assets/' );
+
+		return array(
+			new Script(
+				'googlesitekit-modules-search-console',
+				array(
+					'src'          => $base_url . 'js/googlesitekit-modules-search-console.js',
+					'dependencies' => array(
+						'googlesitekit-vendor',
+						'googlesitekit-api',
+						'googlesitekit-data',
+						'googlesitekit-modules',
+						'googlesitekit-datastore-site',
```

Comment:
This asset does not use the `core/site` store I think, so this dependency should be moved from here.

Reply:
Fixed